src/Bridge/Symfony/DbToolsBundle.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare (strict_types=1);
+
+namespace MakinaCorpus\DbToolsBundle\Bridge\Symfony;
+
+use MakinaCorpus\DbToolsBundle\Bridge\Symfony\DependencyInjection\Compiler\DbToolsPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+use MakinaCorpus\DbToolsBundle\Bridge\Symfony\DependencyInjection\DbToolsExtension;
+
+class DbToolsBundle extends Bundle
```

Comment:
I'm not even sure this would be necessary. The bundle works with zero conf (per default it maps everything on doctrine configuration) the only required thing is to add the bundle into your `config/bundles.php`.

Reply:
OK problem lies in the fact that flex automatically searches for `<your PSR-4>[Bundle]/<your PSR-4>Bundle` named class, but if we put it deeper in `src/Bridge/Symfony` it'll simply don't work anymore... Since we kept a bundle class with the original name for backward compatibility, it's not going to be a problem before 3.0, but I've opened an issue: https://github.com/makinacorpus/DbToolsBundle/issues/168